tests/resources/id_rsa

```diff
@@ -0,0 +1,54 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
maybe rename this file 'dummy' or suchlike, just to stop idiots from scanning our GitHub repo and thinking we were foolish enough to publish a key which is used for anything other than unit tests and acceptance tests.